app/views/visual_groups/edit.html.erb

```diff
@@ -0,0 +1,52 @@
+<%
+  @title = :edit_visual_group_title.t(name: @visual_group.name)
+%>
+
+<p id="notice"><%= notice %></p>
+
+<p>
+  <%= link_to(:visual_group_show.t, @visual_group) %> |
+  <%= link_to(:visual_group_back_to_index.t,
+              visual_model_visual_groups_path(@visual_group.visual_model)) %>
+</p>
+
+<%= render 'form', visual_group: @visual_group %>
+
+<hr>
+
+<p>
+  <strong><%= :visual_group_includes_names.t %>:</strong><br>
+  <% @visual_group.distinct_names.each do |name| %>
+  <%= link_to(name[0],
+              edit_visual_group_path(@visual_group, filter: name[0]))%><br>
+  <% end %>
+</p>
+
+<hr>
+
+<strong><%= :edit_visual_group_filter_options.t %></strong>
+<%= form_with(url: edit_visual_group_path(@visual_group), method: :get) do |form| %>
```

Comment:
Here again i'd add an html `id` to help with the JS selector (and Capybara selector)
```suggestion
<%= form_with(url: edit_visual_group_path(@visual_group), method: :get,
              id: "visual_group_filters_form") do |form| %>
```